Tests/OptimizelyTests-DataModel/ProjectConfigTests.swift

```diff
@@ -80,6 +80,97 @@ class ProjectConfigTests: XCTestCase {
         XCTAssertEqual(featureMap["1004"], ["2002"])
     }
     
+    func testHoldoutIdMapIsBuiltFromProject() {
+        var exp0 = ExperimentTests.sampleData
+        var exp1 = ExperimentTests.sampleData
+        var exp2 = ExperimentTests.sampleData
+        var exp3 = ExperimentTests.sampleData
+        var exp4 = ExperimentTests.sampleData
+        exp0["id"] = "1000"
+        exp1["id"] = "1001"
+        exp2["id"] = "1002"
+        exp3["id"] = "1003"
+        exp4["id"] = "1004"
+       
+        
+        var holdout0 = HoldoutTests.sampleData
```

Comment:
I do not see this sampleData. We can use this sampleData as a guideline for other SDKs.